applications/dashboard/models/class.usermodel.php

```diff
@@ -5229,22 +5229,27 @@ public function clearSectionNavigationPreference($url = '', $userID = '', $reset
         }
 
         $user = $this->getID($userID);
-        $preferences = val('Preferences', $user, []);
-        $landingPages = val('DashboardNav.SectionLandingPages', $preferences, []);
+        $preferences = $user->Preferences ?? [];
+        $landingPages = $preferences['DashboardNav.SectionLandingPages'] ?? [];
+        $sectionPreference = $preferences['DashboardNav.DashboardLandingPage'] ?: '';
```

Comment:
What this a conscious choice to make this one `?:` instead of `??`

Is there some reason we might want the looser check?

Reply:
Not really, I originally wanted to prevent falsy values getting changed unnecessarily, but after looking through the code there isn't really a place where it could be anything other than a string or unset.